### PR TITLE
Added 4 server requests (CRUD) to server's index.js

### PR DESCRIPTION
**Pull request by lancl**
@samueljshih, please take a look when you have time. Thanks!

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -24,8 +31,22 @@ const headerDBSchema = new mongoose.Schema({
   }
 });
 
-const HeaderDB = mongoose.model('HeaderDB', headerDBSchema);
-module.exports = HeaderDB;
+// const HeaderDB = mongoose.model("HeaderDB", headerDBSchema);
```
Comment (samueljshih) [inline database/index.js]: Remove comments if they aren't descriptive.

File: database/index.js [hunk 2]
```diff
@@ -24,8 +31,22 @@ const headerDBSchema = new mongoose.Schema({
...
+  Artists.find({ artistID: artistID }, function(err, artistObj) {
+    if (err) {
+      callback(err, null);
+    } else {
+      callback(null, artistObj);
+    }
+  });
+};
+
+// module.exports.Artists = Artists;
```
Comment (samueljshih) [inline database/index.js:79]: Can be removed too.

File: database/seeder.js [hunk 3]
```diff
@@ -1,38 +1,51 @@
...
     artistID: i,
     artistName: faker.name.findName(),
     followed: Math.round(Math.random()) ? true : false,
-    followersNumber: 'PLACE_HOLDER',
+    followersNumber: "PLACE_HOLDER",
     verified: Math.round(Math.random() * 0.7 + 0.3) ? true : false,
     artistImages: [],
     about: {
-      Biography: faker.lorem.paragraphs(faker.random.number({ min: 5, max: 10 }), '\n\n'),
+      Biography: faker.lorem.paragraphs(
```
Comment (samueljshih) [inline database/seeder.js:14]: does this variable need to be capitalized? could just be biography.

**Review by samueljshih — APPROVED** [review]

File: database/index.js (hunk 2 above)
Comment (lancl) [reply database/index.js:79]: That's right. No need to keep track of history (for this one).

File: database/seeder.js (hunk 3 above)
Comment (lancl) [reply database/seeder.js:14]: Not really. Inherited the upper case from David's codes, hehe.

**Pull request merged**